src/attr.rs

```diff
@@ -170,6 +177,25 @@ pub mod parsing {
         )
         |
         do_parse!(
+            name_and_token_trees: preceded!(
```

Comment:
I had to implement this here instead of in the `meta_item` rule because I needed to use `delimited!` to limit how much `token_trees` parses. Is there a better way?